install.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+install_path=${1:-./uwazi-production/}
+install_path=${install_path%/}
+
+echo "Building production"
+yarn production-build
+
+echo installing into $install_path
+mkdir -p $install_path
+mkdir -p $install_path/custom_uploads
+mkdir -p $install_path/log
+mkdir -p $install_path/temporal_files
+mkdir -p $install_path/uploaded_documents
+
+echo "Copy app files..."
```

Comment:
Previous statements used verbs in the form of "building", but now it's "Copy"? Shoud this be "Copying..."?